src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -326,7 +326,8 @@ protected virtual async Task PostTokenRequestAsync(AdalResultWrapper resultEx)
         {
             var requestParameters = new DictionaryRequestParameters(Resource, ClientKey)
             {
-                { OAuthParameter.ClientInfo, "1" }
+                { OAuthParameter.ClientInfo, "1" },
+                { OAuthParameter.Scope, OAuthValue.ScopeOpenId }
```

Comment:
you may want to add a comment on why this is added (e.g. needed due to cache requirements on the idt).

Reply:
do we add openid for client creds?